board/stm32f4_config.h

```diff
@@ -0,0 +1,34 @@
+#define PANDA
```

Comment:
should we make another sub directory for these like we do with boards? then we can move the startup and linker scripts there too